x-pack/plugin/eql/src/test/java/org/elasticsearch/xpack/eql/planner/QueryFolderFailTests.java

```diff
@@ -17,9 +18,10 @@ public void testPropertyEquationFilterUnsupported() {
     }
 
     public void testPropertyEquationInClauseFilterUnsupported() {
-        QlIllegalArgumentException e = expectThrows(QlIllegalArgumentException.class,
+        VerificationException e = expectThrows(VerificationException.class,
```

Comment:
I noticed that this exception changed. Should this also be the case for `testPropertyEquationFilterUnsupported`?
https://github.com/elastic/elasticsearch/pull/53244/files#diff-3132b6847876733b6fd2161331f9d044L13-R17

Reply:
Not sure I follow - the other test doesn't have a `in` query so I don't think it should.